Source/Project64/UserInterface/Debugger/Breakpoints.cpp

```diff
@@ -64,7 +64,7 @@ bool CBreakpoints::WBPAdd(uint32_t address)
 bool CBreakpoints::AddExecution(uint32_t address, bool bTemporary)
 {
     PreUpdateBP();
-    breakpoints_t::_Pairib res = m_Execution.insert(breakpoint_t::value_type(address, bTemporary));
+    auto res = m_Execution.insert(breakpoint_t::value_type(address, bTemporary));
```

Comment:
this will likely break older versions of VS

Reply:
The "official" building version is VS2015, right?

These C++2011 keywords are supported at least since VS2013. The change is safe :)

Reply:
@project64 Regarding `_Pairib`, it should not be used directly, anyways.
Here is a reply by the main maintainer of the MSVS standard library:

https://twitter.com/StephanTLavavej/status/1162569765774168065

Reply:
it is used that way for android

Reply:
OK, I'll if-def it to not affect versions prior to VS2029/VC14.2.

Reply:
Updated.